sgkit/io/vcf/vcf_reader.py

```diff
@@ -792,6 +796,10 @@ def vcf_to_zarrs(
                 fields=fields,
                 exclude_fields=exclude_fields,
                 field_defs=field_defs,
+                # If a worker dies, dask will redo the tasks on that worker,
+                # which is unnecessary, with this flag the parse will be skipped
+                # if it has already been done.
+                check_if_done=True,
```

Comment:
Should this parameter be propagated all the way back to the public API so that the user can set it?